src/components/seeds/database/index.tsx

```diff
@@ -607,13 +607,15 @@ export default function Database(props: DatabaseProps): JSX.Element {
   const getHeaderButtons = () => (
     <>
       <Box marginLeft={1} display='inline'>
-        <Button
-          id='more-options'
-          icon='menuVertical'
-          onClick={(event) => event && handleClick(event)}
-          priority='secondary'
-          size='medium'
-        />
+        <Tooltip title='More Options'>
```

Comment:
`title={strings.MORE_OPTIONS)` for i18n purposes? and in the other places..

Reply:
Doh!  Of course.  Had that in my head to come back to and forgot.  Thanks for catching!